presto-main/src/main/java/com/facebook/presto/operator/scalar/MapToMapCast.java

```diff
@@ -274,4 +283,15 @@ public static Block mapCast(
         mapBlockBuilder.closeEntry();
         return (Block) toMapType.getObject(mapBlockBuilder, mapBlockBuilder.getPositionCount() - 1);
     }
+
+    public static boolean canSkipDuplicateKeysCheck(Type castFromType, Type castToType)
```

Comment:
also nit: perhaps call them fromType and toType to improve readability (otherwise the difference between the two names is in the middle)

Reply:
Done